src/containers/User/Login.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import './Registration.css';
+import './Login.css';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { withAlert } from 'react-alert';
+import { loginUser } from '../../api/api';
+import { setAuthToken, setUserId } from '../../global/state/globalActions';
+import { connect } from 'react-redux';
+
+export default class Login extends Component {
+   constructor(props) {
+      super(props);
+
+      this.state = {
+         email: '',
+         password: ''
+      };
+   }
+
+   handleInputChange = (e) => {
+      const { name, value } = e.target;
+      this.setState(() => ({ [name]: value }));
+   };
+
+   handleLoginClick = async () => {
+      try {
+         const { email, password } = this.state;
+         const res = await loginUser({ email, password });
+         this.setState({
```

Comment:
Is this clearing the state step necessary?

Reply:
IN